.github/workflows/sycl_update_gpu_driver.yml

```diff
@@ -0,0 +1,49 @@
+name: Update GPU driver
+
+on:
+  schedule:
+    - cron:  '0 3 * * 2'
+
+jobs:
+  update_driver_linux:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(devops/scripts/update_drivers.py linux)"
```

Comment:
I'm not sure this will work. `date +%V` will give you workweek number. Should we use that instead?

Reply:
done. Not sure how to test that it's working correctly.

Reply:
Thanks! The advice on testing on my own repo was clutch!